Changelog.md

```diff
@@ -12,7 +12,8 @@ Compiler Features:
 
 
 Bugfixes:
- * Solidity Upgrade Tool ``solidity-upgrade``: Fix the tool returning success code on uncaught exceptions.
+* Yul Optimizer: Hash hex and decimal literals according to their value instead of their representation.
+* Solidity Upgrade Tool ``solidity-upgrade``: Fix the tool returning success code on uncaught exceptions.
```

Comment:
```suggestion
 * Yul Optimizer: Hash hex and decimal literals according to their value instead of their representation.
 * Solidity Upgrade Tool ``solidity-upgrade``: Fix the tool returning success code on uncaught exceptions.
```
@cameel mentioned that it should in fact be indented by one space, and that the `Compiler Features` section should be fixed.

Reply:
We may also want to include the main effect of the change, i.e. that this may cause more function deduplication - that's what makes this user-facing in the end. ``improving the detection of equivalent functions.`` or sth along nicer those lines.